test/e2e/storage_test.go

```diff
@@ -96,6 +86,37 @@ func testStorage(t *testing.T) {
 			}
 		}()
 	}
+	if inTreeUpgrade {
+		err = tc.labelPodsNode(winServerDeployment.Spec.Selector.MatchLabels, map[string]string{storageTestLabel: "true"})
+		require.NoError(t, err)
```

Comment:
the previous error msg :"error labeling node for upgrade test" seems helpful and can be retained.

Reply:
+1